### PR TITLE
docs: Added cookbook to run arqbackup with minio.

### PR DESCRIPTION
**Pull request by koolhead17**

### CODE REVIEW

**Comment by harshavardhana** [comment]
ping @deekoder

File: docs/arqbackup-minio.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Arq backup with Minio [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```
Comment (harshavardhana) [inline docs/arqbackup-minio.md]: Fix gitter link to slack

File: docs/arqbackup-minio.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Arq backup software is available for Mac and Windows. Arq stores multiple versions of  files in encrypted format. The  backups can be stored on various cloud backends including Minio. This document will walk you through arqbackup configuration with Minio backend.
```
Comment (harshavardhana) [inline docs/arqbackup-minio.md]: a space character between "of  files"

File: docs/arqbackup-minio.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Arq backup with Minio [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/minio/minio?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+Arq backup software is available for Mac and Windows. Arq stores multiple versions of  files in encrypted format. The  backups can be stored on various cloud backends including Minio. This document will walk you through arqbackup configuration with Minio backend.   
+
+## 1. Prerequisites
+
+* [Arq](https://www.arqbackup.com/) backup installed.
+* Minio Server is running on localhost on port 9000 in ``HTTP``, follow [Minio quickstart guide](https://docs.minio.io/docs/minio-quickstart-guide) for installing Minio.
```
Comment (harshavardhana) [inline docs/arqbackup-minio.md]: Reword

* Minio Server is running on localhost:9000, follow [Minio quickstart guide] ...

File: docs/arqbackup-minio.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+## 2. Steps
+
+From choose your backup destination tab select ``Other S3-Comptible service`` and provide Minio server configuration details.
```
Comment (harshavardhana) [inline docs/arqbackup-minio.md]: Reword

"From choose" - From what?

File: docs/arqbackup-minio.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+![D_BUCKET](https://raw.githubusercontent.com/minio/cookbook/master/docs/screenshots/arqbackup.png?raw=true)
+
+Next select an existing bucket for storing backup or let Arq create a storage backup and carry out operations as per need. Check official Arq docs for feature details.
```
Comment (harshavardhana) [inline docs/arqbackup-minio.md]: This sentence doesn't make sense..

**Comment by harshavardhana** [comment]
// cc @koolhead17 - some comments here can you address them?

**Comment by harshavardhana** [comment]
> // cc @koolhead17 - some comments here can you address them?

ping @koolhead17

File: docs/arqbackup-minio.md (hunk 1 above)
Comment (koolhead17) [inline docs/arqbackup-minio.md]: done

File: docs/arqbackup-minio.md (hunk 2 above)
Comment (koolhead17) [inline docs/arqbackup-minio.md]: done.

File: docs/arqbackup-minio.md (hunk 3 above)
Comment (koolhead17) [inline docs/arqbackup-minio.md]: done

File: docs/arqbackup-minio.md (hunk 5 above)
Comment (koolhead17) [inline docs/arqbackup-minio.md]: made is more simple.

File: docs/arqbackup-minio.md (hunk 4 above)
Comment (koolhead17) [inline docs/arqbackup-minio.md]: fixed.

**Pull request merged**